### PR TITLE
Add Janus-IDP Showcase app

### PR DESCRIPTION
**Pull request by PatAKnight**
This PR is to add the Janus-IDP Showcase app to ArgoCD

### CODE REVIEW

**Review by tumido — CHANGES_REQUESTED** [review]
/ok-to-test

LGTM, there's one small issue - a missing link. You need to list the `janus-idp` folder in:

https://github.com/operate-first/apps/blob/master/argocd/overlays/moc-infra/applications/envs/moc/smaug/kustomization.yaml

Otherwise the manifest doesn't get applied.

File: argocd/overlays/moc-infra/applications/envs/moc/smaug/janus-idp/janus-idp.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+aapiVersion: argoproj.io/v1alpha1
```
Comment (tumido) [inline argocd/overlays/moc-infra/applications/envs/moc/smaug/janus-idp/janus-idp.yaml:1]:
```suggestion
apiVersion: argoproj.io/v1alpha1
```

**Review by tumido — CHANGES_REQUESTED** [review]

File: argocd/overlays/moc-infra/applications/envs/moc/smaug/janus-idp/janus-idp.yaml (hunk 1 above)
Comment (PatAKnight) [reply argocd/overlays/moc-infra/applications/envs/moc/smaug/janus-idp/janus-idp.yaml]: Missed that typo, should be fixed now

**Comment by PatAKnight** [comment]
@tumido Added the link to the folder. Hopefully, everything looks good now

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**